### PR TITLE
Prepare the _index.adoc and convert all the MD syntax to AsciiDoc

### PR DESCRIPTION
**Pull request by sergio-carlavilla**
Include all the files in the _index.adoc and convert all the MD syntax to AsciiDoc.

In the _index.adoc there's some files that I don't included because I don't know where should be.

This is the list:

* cbsd.adoc
* ena.adoc
* marvell.adoc
* nv-sndstat.adoc
* pkgbase.live.adoc
* potluck.adoc
* sysctl-improvements.adoc
* virtualbox.adoc

### CODE REVIEW

File: 2021q1/clusteradm.adoc [hunk 1]
```diff
@@ -1,45 +1,45 @@
...
-  * Put local Git mirrors on the universe machines
-  * Replaced disks in the UK mirror
-  * Replaced a disk in pointyhat (https://pkg-status.freebsd.org)
-  * Recycled some old dead-weight servers eating up rackspace and power at our primary cluster site
-  * Upgraded developer reference platforms
-    * ref{11,12,13,14}-{amd64,i386}
-    * universe*
-  * Installed two new aarch64 machines
-    * ref12-aarch64, ref13-aarch64, ref14-aarch64
-    * security-officer aarch64 freebsd-update builder
```
Comment (debdrup) [inline 2021q1/clusteradm.adoc:20]: Same with these entries as the ref and universe machines, these are also sub-items of the aarch64 machines.

**Review by debdrup — CHANGES_REQUESTED** [review]
I noticed a few things:

Why is `#` being converted to `=`, and why are they only used at the start?

Please be consistent about using either double-spacing or ` +` at the end of Links And Contactc - at present, it's a mixed bag.

Also, and this applies for all files, please don't touch the indentation of items and tabs/spacing for Links and Contacts.
It's important for the eventual conversion to plaintext that gets sent out via email.

**Comment by sergio-carlavilla** [comment]
> 
> Why is `#` being converted to `=`, and why are they only used at the start?
> 

This is because an AsciiDoctor heading it's `=` not `#`. AsciiDoctor doesn't understand the MarkDown syntax.

> Please be consistent about using either double-spacing or ` +` at the end of Links And Contactc - at present, it's a mixed bag.
> 
> Also, and this applies for all files, please don't touch the indentation of items and tabs/spacing for Links and Contacts.
> It's important for the eventual conversion to plaintext that gets sent out via email.

We need to use the `+` character to tell AsciiDoctor to break the line.
Also we need to remove the tabs/spacing in all the lists.

**Comment by debdrup** [comment]
> This is because an AsciiDoctor heading it's `=` not `#`. AsciiDoctor doesn't understand the MarkDown syntax.

Fair enough, I can live with that. 

> We need to use the `+` character to tell AsciiDoctor to break the line.
> Also we need to remove the tabs/spacing in all the lists.

If we remove the tab/spacing, the layout of the plaintext email is going to change considerably and be considerably less readable - that's not good, and needs to be solved before this can be merged.

You've also removed indentation for sub-items which are intended to be on the second level of the list, so you at least need to fix that - see lines 16 and 17 as well as 19 and 20 in the clusteradm report.

File: 2021q1/ci.adoc [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
-Link: link:https://ci.FreeBSD.org/hwlab[FreeBSD Hardware Testing Lab]  
-Link: link:https://artifact.ci.FreeBSD.org[FreeBSD CI artifact archive]  
-Link: link:https://hackmd.io/@FreeBSD-CI[FreeBSD CI weekly report]  
-Link: link:https://wiki.freebsd.org/Jenkins[FreeBSD Jenkins wiki]  
-Link: link:https://wiki.freebsd.org/HostedCI[Hosted CI wiki]  
-Link: link:https://wiki.freebsd.org/3rdPartySoftwareCI[3rd Party Software CI]  
-Link: link:https://preview.tinyurl.com/y9maauwg[Tickets related to freebsd-testing@]  
+== Continuous Integration
+
+Link: link:https://ci.FreeBSD.org[FreeBSD Jenkins Instance]  +
```
Comment (debdrup) [inline 2021q1/ci.adoc:3]: There's double-space and a plus here.

File: 2021q1/weblate.adoc [hunk 3]
```diff
@@ -1,22 +1,23 @@
-## FreeBSD Translations on Weblate ##
+== FreeBSD Translations on Weblate
 
-Link: [Translate FreeBSD on Weblate wiki](https://wiki.freebsd.org/DocTranslationOnWeblate)  
-Link: [FreeBSD Weblate Instance](https://translate-dev.freebsd.org/)
+Link: link:https://wiki.freebsd.org/DocTranslationOnWeblate[Translate FreeBSD on Weblate wiki] +
```
Comment (debdrup) [inline 2021q1/weblate.adoc:3]: Here there's just a single space and a plus.

**Comment by sergio-carlavilla** [comment]
> > This is because an AsciiDoctor heading it's `=` not `#`. AsciiDoctor doesn't understand the MarkDown syntax.
> 
> Fair enough, I can live with that.
> 
> > We need to use the `+` character to tell AsciiDoctor to break the line.
> > Also we need to remove the tabs/spacing in all the lists.
> 
> If we remove the tab/spacing, the layout of the plaintext email is going to change considerably and be considerably less readable - that's not good, and needs to be solved before this can be merged.
> 
Ok, let me check this
> You've also removed indentation for sub-items which are intended to be on the second level of the list, so you at least need to fix that - see lines 16 and 17 as well as 19 and 20 in the clusteradm report.

Let me check this too

**Comment by debdrup** [comment]
Actually, let me just get this correct - is the actual text, that's displayed as html, still indented when using list items, even if the adoc source code isn't indented?
Because the way I generate the plaintext for the email is with a text browser that outputs things to html at a given width (72 characters, to leave room for indenting in case of quotes), and I think it'll look fine as long as the generated html looks fine.

**Comment by alfonsosiciliano** [comment]
@sergio-carlavilla 

> In the _index.adoc there's some files that I don't included because I don't know where should be.
 
"Kernel": ena.adoc, nv-sndstat.adoc, marvell.adoc
"Projects": virtualbox.adoc
"Miscellaneous": sysctl-improvements.adoc (or kernel, I am doubtful)
"Third-Party Projects": potluck.adoc, cbsd.adoc, pkgbase.live.adoc

@debdrup I found some typo but I have not write access here so I' ll open a pull request after this commit.

**Comment by debdrup** [comment]
I'm gonna go ahead and fix the things in a local checkout of this PR, and then rebase it onto main, and merge it to main. That way we can move forward if there's anything else that needs to happen.

**Comment by sergio-carlavilla** [comment]
> 
> 
> I'm gonna go ahead and fix the things in a local checkout of this PR, and then rebase it onto main, and merge it to main. That way we can move forward if there's anything else that needs to happen.

Perfect, right now I'm in my job and I can't help.

With the list, if you want to add subelements use **

For example:

* Element 1
** Subelement 1.1
* Element 2

**Comment by debdrup** [comment]
> Perfect, right now I'm in my job and I can't help.
> 
> With the list, if you want to add subelements use **
> 
> For example:
> 
>     * Element 1
>       ** Subelement 1.1
> 
>     * Element 2

Alright, I'll fix the few things that need changing, so feel free to open a new PR with what needs to be fixed.

I'm closing this now, as I believe everything has been addressed.

**Pull request closed without merging**